src/game/models/mod.rs

```diff
@@ -2,157 +2,17 @@
 
 use std::f64;
 use std::f64::consts::PI;
-use std::ops::{Add, AddAssign, Rem, RemAssign, Sub, SubAssign, Div, DivAssign};
 
 use opengl_graphics::GlGraphics;
-use piston_window::{Context, UpdateArgs, Size};
-use rand;
+use piston_window::{Context, UpdateArgs};
 
 pub mod player;
```

Comment:
Since these were already out of order before this PR, should I do it in a separate one, or do it in here? Eh, I'll do it in here.